bolt-sidecar/src/state/execution.rs

```diff
@@ -107,6 +114,11 @@ impl<C: StateFetcher> ExecutionState<C> {
         &self.block_templates
     }
 
+    /// Returns the chain ID of the chain
+    pub fn chain_id(&self) -> u64 {
+        self.chain_id
+    }
```

Comment:
why is this needed? can't we just use `self.chain_id` in `try_commit` below?

Reply:
right I just thought to have a getter fn but on another thought, it might not be useful, removed